### DISCUSSION TITLE
Keybind for pausing / starting music in the game menu

### DISCUSSION THREAD

**Discussion by Temanor** [ppy/osu#21632] [Ideas] [answered]
In osu!stable, you can toggle music playing by pressing C on the keyboard, without manually moving the mouse over to click pause.
In the stable version, this wasn't really a huge deal as if you were to tab out of the game, the music would stop, but in lazer it just lowers the volume.
This was a feature I didn't really realise I used this much until I started using lazer.

**Comment by ghost** [comment, marked as answer]
by default, this keybind is f3

**Reply by peppy** [reply]
You can change this key binding in settings. You can also change the volume of the game fade when inactive (ie. set to zero for the behaviour you're looking for):


https://user-images.githubusercontent.com/191335/207492921-eac1a469-b958-4cb3-b0b2-00cc23a1a4e8.mp4



But to correct you: stable should not mute when you alt-tab out. If that's happening it's likely something else on your system doing that.

**Reply by Temanor** [reply]
Oh... I was searching for it but couldn't find it
And I wasn't quite clear, but when I said tab out, I meant doing Windows+D

**Reply by peppy** [reply]
That case should also be covered.

**Reply by Temanor** [reply]
Nevermind, I am just stupid 😓
I got it mixed up with the pause sound in game.
When you pause in stable and tab out, it gets muted.
When you pause in lazer and tab out, it just gets lowered.
